articles/api/management/v2/user-search.md

```diff
@@ -3,44 +3,49 @@ description: This page lists several examples of user search queries using query
 section: apis
 crews: crew-2
 ---
-
 # User Search
 
-As an Auth0 admin you can search different fields of your users using [query string syntax](/api/management/v2/query-string-syntax) with the Users section of the Management API.
+Auth0 allows you, as an administrator, to search for users using [Lucene Query Syntax](http://www.lucenetutorial.com/lucene-query-syntax.html). This syntax can be used either in the [Users section of the Auth0 Dashboard](${manage_url}/#/users) or with the [`GET /api/v2/users` endpoint](/api/management/v2#!/Users/get_users) of the [Management API](/api/v2)  
 
-The `user_metadata`, `app_metadata` or the [normalized user profile](/user-profile/normalized) fields are searchable. Note that users have read/write access to the `user_metadata` field but only read-only access to the `app_metadata` field.
+This document will demonstrate the various ways in which you can search for users, as well as give some example queries. It is however suggested that you also reference the [Query String Syntax document](/api/management/v2/query-string-syntax) for more examples of the query string syntax.
 
-Auth0 User Search uses [Lucene Query Syntax](http://www.lucenetutorial.com/lucene-query-syntax.html).
+All the [normalized user profile](/user-profile/normalized) fields, as well as the `user_metadata` and `app_metadata` are searchable.
 
 ::: note
```

Comment:
This note seems a bit out of context. Maybe move it to a paragraph `Sorting Results` after the `Search using Postman` paragraph? Also maybe add info on how to specify the default sort field?